website/content/docs/api-gateway/configuration/gateway.mdx

```diff
@@ -0,0 +1,316 @@
+---
+layout: docs
+page_title: Consul API Gateway Gateway
+description:  >-
+  Consul API Gateway Gateway
+---
+
+# Gateway
```

Comment:
This particular page needs a bit of help on formatting - the tables should likely use inline `code` formatting for parameters and have all columns left-aligned, and the Configuration Model section (and possibly the individual sections under Specification?) could maybe benefit from formatting similar to https://www.consul.io/docs/k8s/helm to make it easier to scan.

Reply:
I think the table should be all left-aligned. The different alignment in each column is kinda jarring.

<img width="943" alt="image" src="https://user-images.githubusercontent.com/7556325/180494559-45aa27ee-e514-4168-844e-5854793db656.png">

While I agree that this new config model is easier to scan, we should update the links so they jump to the correct section -- that may lead to a better experience.